.github/renovate.json5

```
@@ -44,6 +44,7 @@
       ],
       matchStrings: [
         "depName=(?<depName>.*?)\\s.*?_version=(?<currentValue>.*)\\s",
```

```suggestion
        "depName=(?<depName>.*?)\\s.*?_version=(?<currentValue>.*)\\s",
        "depName=(?<depName>.*?)\\s.*?_VERSION=(?<currentValue>.*)\\s",
```
What about this, instead of renaming the variables in the script? IMO it's a better format to have the constants in capital letters

---

I'm okey to do it this way as well.

---

@albertogdd sorry it won't work, after quick look we need to change all files where we use this pattern:
- for example hack/build/ci/third-party-licenses.sh and it's not make file

---

But can't we keep both regexes?

---

ohhh, i don't want to maintain 2 regular expressions for very similar stuff.

---

changing a whole makefile for a single variable in a shell script seems like a bad trade

---

fixed in 282d77e90